.github/workflows/development.yml

```diff
@@ -0,0 +1,47 @@
+name: Test Builds On Merge Creation
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  testbuild:
+    name: Test Compile On Pull Request
+    runs-on: ubuntu-22.04
+    env:
+      OUTPUT_PATH: output/get5
+      OUTPUT_SM_PATH: output/get5/addons/sourcemod
+      SM_JSON_INC_PATH: get5/dependencies/sm-json/addons/sourcemod/scripting/include
+      STEAMWORKS_URL: https://raw.githubusercontent.com/KyleSanderson/SteamWorks/1.2.3c/Pawn/includes/SteamWorks.inc
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: 'get5'
+        submodules: true
+
+    - uses: rumblefrog/setup-sp@master
+      with:
+        version: '1.10.x'
+
+    - name: Get shorthand commit.
+      id: vars
+      run: |
+        echo "::set-output name=sha_short::$(echo ${{ github.sha }} | cut -c1-6)"
+        echo "::set-output name=get5_version::$(cat get5/scripting/get5/version.sp | grep -Eo \"[0-9]*\.[0-9]\.[0-9]-[a-zA-z]{3}\")"
```

Comment:
Regex doesn't match stuff like `0.12.12-dev`

Needs to be `[0-9]+\.[0-9]+\.[0-9]+-[a-zA-Z]+` - no reason to lock it into 3 symbols at the end either